prompts.py

```diff
@@ -1,18 +1,19 @@
 from lang import lang, all_langs
 
+proof_marker = None
+check_proof = None
 if lang == 'Dafny':
     proof_marker = 'ensures'
 elif lang == 'Coq':
     proof_marker = 'Qed'
 elif lang == 'Lean4':
-    proof_marker = None
-else:
-    proof_marker = None
+    check_proof = lambda v: v.count('theorem') > 2 # because of example
```

Comment:
I copied the example from https://github.com/yangky11/lean4-example/blob/main/Lean4Example.lean, it contains two proofs for the sake of it, but most proofs should only have 1 "theorem." I think a check for "theorem" in "v" should work (provided we limit the problems to proofs)!

Reply:
Indeed, so > 2 means 1 new theorem!

Reply:
Oh great! This should work then.